### ISSUE TITLE
Running pipeline-tutorial on openshift 4.3.1

### ISSUE THREAD

**Issue by kevinyu98** [openshift/pipelines-tutorial#66]
OpenShift Version
4.3.1
Installed
openshift-pipelines-operator.v0.10.7

on ibm cloud

when I follow this example

pipeline-tutorial

after started the pipeline run, when I try to get the logs
```
[build-ui : build] STEP 2: LABEL "io.openshift.s2i.build.image"="registry.access.redhat.com/rhscl/python-36-rhel7" "io.openshift.s2i.build.source-location"="."
[build-ui : build] error building at STEP "LABEL "io.openshift.s2i.build.image" "registry.access.redhat.com/rhscl/python-36-rhel7" "io.openshift.s2i.build.source-location" "."": error ensuring container path "/opt/app-root/src": lstat /var/lib/containers/storage/overlay/262cf5f36861100bf27c9eb5f9baa42f2471984ee129a5f1dd901d81cf3d4cbd/merged/opt: invalid argument

failed to get logs for task build-ui : container step-build has failed : [{"name":"","digest":"","key":"StartedAt","value":"2020-03-18T05:00:49Z","resourceRef":{}}]`

the detail logs:
`tkn pipelinerun logs build-and-deploy-run-wcp2z -f -n pipelines-tutorial
(base) Qianyangs-MBP:kevin-openshift-v4 qianyangyu$ tkn pipeline list
NAME AGE LAST RUN STARTED DURATION STATUS
build-and-deploy 18 minutes ago build-and-deploy-run-wcp2z 1 minute ago --- Running
(base) Qianyangs-MBP:kevin-openshift-v4 qianyangyu$ tkn pipeline logs -f
....

[build-api : git-source-api-repo-zjvxx] {"level":"info","ts":1584507618.453761,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: "KO_DATA_PATH" does not exist or is empty"}
[build-api : git-source-api-repo-zjvxx] {"level":"info","ts":1584507646.4758844,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: "KO_DATA_PATH" does not exist or is empty"}
[build-api : git-source-api-repo-zjvxx] {"level":"info","ts":1584507648.5243325,"logger":"fallback-logger","caller":"git/git.go:102","msg":"Successfully cloned http://github.com/openshift-pipelines/vote-api.git @ master in path /workspace/source"}
[build-api : git-source-api-repo-zjvxx] {"level":"warn","ts":1584507648.5244727,"logger":"fallback-logger","caller":"git/git.go:149","msg":"Unexpected error: creating symlink: symlink /tekton/home/.ssh /root/.ssh: file exists"}
[build-api : git-source-api-repo-zjvxx] {"level":"info","ts":1584507648.6062376,"logger":"fallback-logger","caller":"git/git.go:130","msg":"Successfully initialized and updated submodules in path /workspace/source"}

[build-ui : generate] {"level":"info","ts":1584507622.4569638,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: "KO_DATA_PATH" does not exist or is empty"}
[build-ui : generate] Application dockerfile generated in /gen-source/Dockerfile.gen

[build-api : build] {"level":"info","ts":1584507632.1757002,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: "KO_DATA_PATH" does not exist or is empty"}
[build-api : build] STEP 1: FROM golang:alpine AS builder
[build-api : build] Getting image source signatures
[build-api : build] Copying blob sha256:d909eff282003e2d64af08633f4ae58f8cab4efc0a83b86579b4bbcb0ac90956
[build-api : build] Copying blob sha256:cbb0d8da1b304e1b4f86e0a2fb11185850170e41986ce261dc30ac043c6a4e55
[build-api : build] Copying blob sha256:a50ef8b76e536c1f848f61399fe1e8721531496a1a3501124e2b24f4677f0cd0
[build-api : build] Copying blob sha256:c9b1b535fdd91a9855fb7f82348177e5f019329a58c53c47272962dd60f71fc9
[build-api : build] Copying blob sha256:8b9d9d6824f5457e80af26521acf1c1e52493e7a72889d778eb9bcc5f7eb68c4
[build-api : build] Copying config sha256:51e47ee4db586c983e61a925bea3b7b08f2d7b95718e3bd3fac3da97c1c6325f
[build-api : build] Writing manifest to image destination
[build-api : build] Storing signatures
[build-api : build] STEP 2: WORKDIR /build
[build-api : build] 4b4d17ba12c7cfe122a9b3ed8a5a3687a280d6761547aee55586251584af5321
[build-api : build] STEP 3: ADD . /build/
[build-api : build] 2cb4b995b852e4f1aa3a2fa736d64ad88c8c6b668c53a9b5903f70cf9bf45ce2
[build-api : build] STEP 4: RUN GOOS=linux GARCH=amd64 CGO_ENABLED=0 go build -mod=vendor -o api-server .
[build-ui : build] {"level":"info","ts":1584507640.7401912,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: "KO_DATA_PATH" does not exist or is empty"}
[build-ui : build] STEP 1: FROM registry.access.redhat.com/rhscl/python-36-rhel7
[build-ui : build] Getting image source signatures
[build-ui : build] Copying blob sha256:6a4fa4bc2d06e942c0e92d69614b4ee30c2d409c95f29a3a22ece8087ce164be
[build-ui : build] Copying blob sha256:455ea8ab06218495bbbcb14b750a0d644897b24f8c5dcf9e8698e27882583412
[build-ui : build] Copying blob sha256:bb13d92caffa705f32b8a7f9f661e07ddede310c6ccfa78fb53a49539740e29b
[build-ui : build] Copying blob sha256:c8106f599d69375cbfc2ef44b11812ddc33938ab1e94860b02c262118f837611
[build-ui : build] Copying blob sha256:84e620d0abe585d05a7bed55144af0bc5efe083aed05eac1e88922034ddf1ed2
[build-ui : build] Copying config sha256:3c93c53ba3715f62aad12366410a1cd57957c39f573c0681807000d12f3cccdc
[build-ui : build] Writing manifest to image destination
[build-ui : build] Storing signatures
[build-ui : build] STEP 2: LABEL "io.openshift.s2i.build.image"="registry.access.redhat.com/rhscl/python-36-rhel7" "io.openshift.s2i.build.source-location"="."
[build-ui : build] error building at STEP "LABEL "io.openshift.s2i.build.image" "registry.access.redhat.com/rhscl/python-36-rhel7" "io.openshift.s2i.build.source-location" "."": error ensuring container path "/opt/app-root/src": lstat /var/lib/containers/storage/overlay/262cf5f36861100bf27c9eb5f9baa42f2471984ee129a5f1dd901d81cf3d4cbd/merged/opt: invalid argument

failed to get logs for task build-ui : container step-build has failed : [{"name":"","digest":"","key":"StartedAt","value":"2020-03-18T05:00:49Z","resourceRef":{}}]
[build-api : build] f72f609911547c6bf2364e63cf786987cd7189c74d787d585d659284b7feba77
[build-api : build] STEP 5: FROM scratch
[build-api : build] STEP 6: WORKDIR /app

```

a related issue

**Comment by nastacio** [comment]
Had the exact same problem. 

The Dockerfile is fine and can be built locally, so I narrowed it down to a filesystem access problem.

Try this, solved my problem: https://github.com/containers/buildah/issues/2198
I applied it to the "buildah" ClusterTask, adding an extra parameter to the "buildah bud ..." call.

![image](https://user-images.githubusercontent.com/3278623/82391586-a2847c80-9a0f-11ea-827c-67114e3d3a71.png)
